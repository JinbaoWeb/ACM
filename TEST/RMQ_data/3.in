41 17
6334 26500 19169 15724 11478 29358 26962 24464 5705 28145 23281 16827 9961 491 2995 11942 4827 5436 32391 14604 3902 153 292 12382 17421 18716 19718 19895 5447 21726 14771 11538 1869 19912 25667 26299 17035 9894 28703 23811 31322
3 4
10 26
26 31
1 9
12 20
4 35
16 17
5 16
23 39
22 31
8 26
18 36
15 31
8 37
10 21
0 19
4 28
