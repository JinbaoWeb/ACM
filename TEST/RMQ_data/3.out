15724
19169
153
32391
5447
21726
5705
29358
491
32391
153
32391
4827
11942
491
29358
292
28703
153
21726
153
32391
153
32391
153
32391
153
32391
491
32391
0
32391
153
32391
